### PR TITLE
MOM 5 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/drana_and_linvala.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Drana and Linvala
+ManaCost:1 W W B
+Types:Legendary Creature Vampire Angel
+PT:3/4
+K:Flying
+K:Vigilance
+S:Mode$ CantBeActivated | Activator$ Opponent | AffectedZone$ Battlefield | ValidCard$ Creature | ValidSA$ Activated | Description$ Activated abilities of creatures your opponents control can't be activated.
+S:Mode$ Continuous | Affected$ Card.Self | EffectZone$ Battlefield | GainsAbilitiesOf$ Creature.OppCtrl | GainsValidAbilities$ Activated | GainsAbilitiesActivateIgnoreColor$ AnyType->AnyColor | Description$ CARDNAME has all activated abilities of all creatures your opponents control. You may spend mana as though it were mana of any color to activate those abilities.
+SVar:PlayMain1:TRUE
+Oracle:Flying, vigilance\nAzctivated abilities of creatures your opponents control can't be activated.\nDrana and Linvala has all activated abilities of all creatures your opponents control. You may spend mana as though it were mana of any color to activate those abilities.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/drana_and_linvala.txt:10]: Typo

File: forge-gui/res/cardsfolder/upcoming/thalia_and_the_gitrog_monster.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Name:Thalia and The Gitrog Monster
+ManaCost:1 W B G
+Types:Legendary Creature Human Frog Horror
+PT:4/4
+K:First Strike
+K:Deathtouch
+S:Mode$ Continuous | Affected$ You | AdjustLandPlays$ 1 | Description$ You may play an additional land on each of your turns.
+R:Event$ Moved | ValidCard$ Creature.OppCtrl,Land.nonBasic+OppCtrl | Destination$ Battlefield | ReplaceWith$ ETBTapped | ActiveZones$ Battlefield | Description$ Creatures and nonbasic lands your opponents control enter the battlefield tapped.
+SVar:ETBTapped:DB$ ChangeZone | Origin$ All | Destination$ Battlefield | Tapped$ True | Defined$ ReplacedCard
```
[the remark this answers is not in the document]
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/thalia_and_the_gitrog_monster.txt:9]: Hmm I copied this from the OG thalia. I'll take a look

[the remark this answers is not in the document]
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/thalia_and_the_gitrog_monster.txt:9]: Then your version is quite outdated, please use recent scripts for templating

[the remark this answers is not in the document]
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/thalia_and_the_gitrog_monster.txt:9]: Still not correct

File: forge-gui/res/cardsfolder/upcoming/ghalta_and_mavren.txt [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+Types:Legendary Creature Dinosaur Vampire
+PT:12/12
+K:Trample
+T:Mode$ AttackersDeclared | AttackingPlayer$ You | Execute$ TrigCharm | TriggerDescription$ Whenever you attack, ABILITY
+SVar:TrigCharm:DB$ Charm | CharmNum$ 1 | Choices$ DBDino,DBVampire
+SVar:DBDino:DB$ Token | TokenAmount$ 1 | TokenOwner$ You | TokenScript$ g_x_x_dinosaur_trample | TokenPower$ X | TokenToughness$ X | TokenTapped$ True | TokenAttacking$ True | TokenOwner$ You | SpellDescription$ Create a tapped and attacking X/X green Dinosaur creature token with trample, where X is the greatest power among other attacking creatures.
+SVar:DBVampire:DB$ Token | TokenAmount$ Y | TokenScript$ w_1_1_vampire_lifelink | TokenOwner$ You | SpellDescription$ Create X 1/1 white Vampire creature tokens with lifelink, where X is the number of other attacking creatures.
+SVar:X:Count$Valid Creature.YouCtrl+Other+attacking$GreatestPower
+SVar:Y:Count$Valid Creature.attacking+Other
+DeckHas:Ability$LifeGain|Token
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/ghalta_and_mavren.txt:12]: add the hasattackeffect

**Review by Northmoc — APPROVED** [review]
Nice 👍

**Pull request merged**